keps/sig-node/606-compute-device-assignment/kep.yaml

```diff
@@ -6,7 +6,7 @@ authors:
   - "@renaudwastaken"
 owning-sig: sig-node
 participating-sigs: []
-status: implementable
+status: implemented
```

Comment:
the PR to lock the feature gate was merged to 1.28. Based on this comment https://github.com/kubernetes/enhancements/pull/4043#issuecomment-1570601071 we can merge it now. I'm fine either way.

Reply:
Thanks @wojtek-t for the clarification. The work is GA in 1.28 . We already merged the change because all the actual work happened in the 1.27 cycle. The only pending item left was to [lock the FG](https://github.com/kubernetes/kubernetes/pull/116525), which missed the deadline because of the long merge queue on the last day of the 1.27 code freeze. I don't mind closing this PR and resubmitting few months in the future.

Reply:
> Based on this comment https://github.com/kubernetes/enhancements/pull/4043#issuecomment-1570601071 we can merge it now.

I don't fully agree here.
I really hope it won't be the case here, but I've seen at least a few cases where something was claimed done and then reverted later and finally missed the release.
So while the chances of that in this cases are super low, I still believe we should actually not workaround the process that we have and mark the KEPs as implemented only after the release in which the feature went stable is actually already released. [So in this case, when 1.28 is out.]

Doing it differently in different cases is simply confusing people around - clear rules always help in my opinion.

Reply:
Sorry for the confusion. I addressed the doubt regarding whether the enhancement will be removed from tracking board if this PR was merged and the issue closes, but I deduced by looking at this PR title that the stage of this enhancement is now being moved to `stable` for 1.28 release which I see is not the case. The status, in case of stable enhancements, should be set to `implemented` only after the release.

Reply:
Sorry for the confusion. I addressed the doubt regarding whether the enhancement will be removed from tracking board if this PR was merged and the issue closes. The status, in case of stable enhancements, should be set to `implemented` only after the release.

Reply:
Sorry for the confusion. I addressed the doubt of whether the enhancement will be removed from tracking board if this PR was merged and the issue closes, but I deduced by looking at the PR title that this PR is changing its stage to `stable` for 1.28 release which I now see is not the case. The status, in case of stable enhancements, should be set to `implemented` only after the release.